README.md

```diff
@@ -1,5 +1,55 @@
 # Skywalking Grafana Plugins
 
+SkyWalking Grafana Plugins provides an extention for [Apache SkyWalking](https://skywalking.apache.org/) visualizting telemetry data on Grafana. 
+
+Currently, SkyWalking supports [PromQL](https://skywalking.apache.org/docs/main/next/en/api/promql-service/) to establish [Grafana metrics dashboard](https://skywalking.apache.org/docs/main/next/en/setup/backend/ui-grafana/). This plugin is focusing on more telemetry data, Service Map Topology.
+
+This plugin provides a Grafana data source implementation.
+
+## Install the latest version of SkyWalking data source in your Grafana application
+
+* Require Grafana 9.5.1
+
+```bash
+> grafana-cli plugins install skywalking-skywalking-datasource
```

Comment:
Why do we have duplicate skywalking prefixes?

Reply:
It is automatically generated during project initialization, based on name, type, and author name in `plugin.json`.